tests/core/resource/test_filter.py

```diff
@@ -8,13 +8,18 @@ class FilteringTest(BaseTest):
     def setUp(self):
         super(FilteringTest, self).setUp()
         self.patch_known_field.start()
-        for i in range(6):
```

Comment:
very cryptic to have the state of the world in a for loop for our unit tests. do you agree that it benefits us to have a nicely readable and solid list of records instead?